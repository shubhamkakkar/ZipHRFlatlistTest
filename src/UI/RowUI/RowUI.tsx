import React from 'react';
import { View } from 'react-native';
import styles from './styleSheetRowUI';

export default function RowUI({ children }: { children: React.ReactNode }) {
  return <View style={styles.rowUIContainer}>{children}</View>;
}
