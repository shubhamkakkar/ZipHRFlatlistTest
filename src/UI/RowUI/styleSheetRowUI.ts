import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  rowUIContainer: {
    flex: 1,
    flexDirection: 'row',
    flexWrap: 'wrap',
  },
});

export default styles;
