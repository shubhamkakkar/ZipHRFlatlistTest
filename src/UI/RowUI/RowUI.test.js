import React from 'react';
import renderer from 'react-test-renderer';
import { View } from 'react-native';
import RowUI from './RowUI';

it('RowUI matches snapshot', () => {
  const tree = renderer
    .create(
      <RowUI>
        <View />
        <View />
      </RowUI>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
