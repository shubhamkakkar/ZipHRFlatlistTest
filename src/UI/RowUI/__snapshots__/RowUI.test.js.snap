// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RowUI matches snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
      "flexWrap": "wrap",
    }
  }
>
  <View />
  <View />
</View>
`;
